test-requirements.txt

```diff
@@ -6,6 +6,7 @@ hacking<0.11,>=0.10.0
 
 coverage>=3.6
 python-subunit>=0.0.18
+retry>=0.9.2
```

Comment:
OpenStack seems to use "retrying", https://github.com/openstack/requirements/blob/master/upper-constraints.txt#L335

I wonder if it's the same thing?